samples/gcp-microservices-demo/README.md

```
@@ -76,7 +76,7 @@ This will deploy all the microservices using official Google Container Registry
 Access the frontend application in your browser:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)